packages/turf-shortest-path/index.ts

```diff
@@ -40,104 +49,103 @@ import { Graph, astar } from "./lib/javascript-astar";
  * //addToMap
  * var addToMap = [start, end, options.obstacles, path];
  */
-function shortestPath(start, end, options) {
+function shortestPath(
+  start: Coord,
+  end: Coord,
+  options: {
+    obstacles?: Polygon | Feature<Polygon> | FeatureCollection<Polygon>;
```

Comment:
I'm kind of surprised that this doesn't also allow MultiPolygons from the JSDoc above where it accepts `Feature`, but this matches the previous types from the deleted .d.ts.

Reply:
Yeah, I haven't pulled too much at loose threads like this 😅